agent_server.py

```
@@ -526,6 +578,70 @@ async def _run_processor():
     logger.info(f"[MCP] Started processor task {task_id} for {lanlan_name}")
     return {"success": True, "task_id": task_id, "status": info["status"], "start_time": info["start_time"]}
 
+# 插件直接触发路由（放在顶层，确保不在其它函数体内）
```

Ruff (0.14.8)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)